pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -754,7 +781,7 @@ func TestRegistriesValidation(t *testing.T) {
 	for _, test := range successTests {
 		imgcfg := newImageConfig(test.name, test.config)
 		cvcfg := newClusterVersionConfig("version", "blah.io/myuser/myimage:test")
-		insecure, blocked, err := getValidRegistries(&cvcfg.Status, &imgcfg.Spec)
+		insecure, blocked, _, err := getValidRegistries(&cvcfg.Status, &imgcfg.Spec)
```

Comment:
Non-blocking: Add actual tests for `allowedRegs` here? Or simplify by not returning that value, maybe, see elsewhere.

Reply:
Changed this function to only validate blocked registries, got rid of the insecure and allowed. Fixed